go/common/enclave.go

```diff
@@ -65,9 +65,6 @@ type Enclave interface {
 	// GetTransactionReceipt returns a transaction receipt given its signed hash, or nil if the transaction is unknown
 	GetTransactionReceipt(encryptedParams EncryptedParamsGetTxReceipt) (EncryptedResponseGetTxReceipt, error)
 
-	// GetRollup returns the rollup with the given hash, or nil if no such rollup exists.
-	GetRollup(rollupHash L2RootHash) (*ExtRollup, error)
```

Comment:
Not needed anymore. I store this on the host side instead, to remove the need for a round trip.